card/mod/05_standard/lib/file_uploader.rb

```diff
@@ -57,8 +57,8 @@ def filename
   def extension
     if file && file.extension.present?
       ".#{file.extension}"
-    elsif original_filename
-      File.extname(original_filename)
+#    elsif original_filename
+#      File.extname(original_filename)
```

Comment:
there used to be cases where this was the only way to get the extension but could be that this is no longer necessary